static/css/main.css

```diff
@@ -78,10 +78,10 @@ html,body
 
 .fa-openhub {
   display: inline-block;
-  height: 24px;
-  width: 24px;
-  background-image: url("https://raw.githubusercontent.com/blackducksoftware/ohloh-ui/master/public/images/openhub_twitter_24.png");
-  filter: brightness(70%) contrast(200%) grayscale(90%);
+  height: 32px;
+  width: 32px;
+  background-image: url("../oh_icon.png");
```

Comment:
you need to put it in static folder

Reply:
also, is it possible to get any permalink for this icon?

Reply:
The icon is already in the static folder, it works fine
I can make it fetch from a permalink if you'd like, but I don't see why that's necessary

Reply:
most of our static content (like fomtawesome icons) are served throught external sources only. 

I that is better for performance reasons and I prefer to avoid adding to static unless necessary

Reply:
Alright, i'll host it on imgur (unless you have any preferences as to where?) @manu-chroma